packages/web-components/src/rating-display/rating-display.styles.ts

```diff
@@ -0,0 +1,136 @@
+import { css } from '@microsoft/fast-element';
+import { display, forcedColorsStylesheetBehavior } from '../utils/index.js';
+import {
+  colorBrandBackground,
+  colorBrandStroke2,
+  colorNeutralBackground1Pressed,
+  colorNeutralForeground1,
+  colorPaletteMarigoldBackground2,
+  colorPaletteMarigoldBackground3,
+  fontFamilyBase,
+  fontSizeBase200,
+  fontSizeBase300,
+  fontWeightSemibold,
+  lineHeightBase200,
+  lineHeightBase300,
+  spacingHorizontalSNudge,
+  spacingHorizontalXS,
+  spacingHorizontalXXS,
+} from '../theme/design-tokens.js';
+import { RatingDisplayColor, RatingDisplaySize } from './rating-display.options.js';
+
+/**
+ * The styles for the Rating Display component.
+ *
+ * @public
+ */
+export const styles = css`
+  ${display('inline-flex')}
+
+  :host {
+    --icon-size: 16px;
+    align-items: center;
+    color: ${colorNeutralForeground1};
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase200};
+    line-height: ${lineHeightBase200};
+    contain: layout style;
+    user-select: none;
+  }
+
+  :host([size=${RatingDisplaySize.small}]) {
+    --icon-size: 12px;
+  }
+
+  :host([size=${RatingDisplaySize.large}]) {
```

Comment:
Rather than tie to the attribute, I'd suggest we leverage custom states with `toggleState` and use the reusable state constants `states/index.ts` as the selector. That will default to `:state()` where supported and `[state--foo]` attribute fallback when not

Reply:
Changed to use custom states and added a couple of new states to `states/index.ts`.